### PR TITLE
[change] Updated screenshots to reflect latest version #312

### LINKED ISSUE #312: [docs] Update screenshots
Let's update the screenshots to reflect the latest versions of the themes.

### PR DESCRIPTION
**Pull request by codesankalp**
Closes #312

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -282,7 +282,7 @@ A general live API documentation (following the OpenAPI specification) at ``/api
 Browsable web interface
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-.. image:: https://github.com/openwisp/openwisp-users/raw/master/docs/images/api-ui.png
+.. image:: https://github.com/openwisp/openwisp-users/raw/docs/docs/images/api-ui.png
```
Comment (nemesisdesign) [inline README.rst:285]: I have one minor request here, can we show something more meaningful here? Eg: the output of /api/v1/users/user/, showing some users.

Before the only endpoint was this one, now we have more endpoints and we can show something better than this.

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (codesankalp) [reply README.rst:285]: Please See: https://github.com/openwisp/openwisp-users/blob/docs/docs/images/api-ui.png

Comment (codesankalp) [reply README.rst:285]: Please see: https://raw.githubusercontent.com/openwisp/openwisp-users/docs/docs/images/api-ui.png

**Review by nemesisdesign — APPROVED** [review]
Thanks!

**Pull request merged**